policyenforcer.go

```diff
@@ -21,6 +21,18 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+// PolicyDecision is the decision made by the policy enforcer.
+type PolicyDecision int
+
+const (
+	// Undertermined is the initial or a temporary state of the policy decision.
+	Undetermined PolicyDecision = iota
+	// Deny is the final state when the policy evaluation is failed.
+	Deny
+	// Allow is the final state when the policy evaluation is succeeded.
+	Allow
+)
```

Comment:
Please add `PolicyDecision` prefix to those enums for naming convention. e.g. `PolicyDecisionAllow`.